.github/workflows/tests.yml

```diff
@@ -20,6 +20,11 @@ jobs:
           '--use-libraries',
           '--use-static-frameworks'
         ]
+        include:
+          - sdk: 'macosx'
+            destination: 'platform=OS X,arch=x86_64'
+          - sdk: 'iphonesimulator'
```

Comment:
This is generating a workflow file syntax error:
https://github.com/google/GoogleSignIn-iOS/actions/runs/1954866506/workflow

Reply:
Done.